requirements.txt

```diff
@@ -33,3 +33,9 @@ django-sendfile
 django-polymorphic
 networkx
 rdflib>=4.1.0
+celery[redis]
+django-celery
+scikit-learn
+python-redis
```

Comment:
oops, I think it should be django-redis! You can try eliminating it, and then when you test celery the error message actually specifies the module name.